packages/docs-website/docs/protocol-tutorial/execute-state-transitions.md

```diff
@@ -107,9 +107,23 @@ As a state channel participant, it is advised to check the `FixedPart` of any ch
 
 - `chainId` -- This needs to match the id of the chain where assets are to be locked. The root of the funding tree for this channel.
 - `participants` -- This should have length at least 2, but no more than 255, and include a public key (account) that you control. Each entry should be a nonzero ethereum address.
-- `channelNonce` -- This should be different to any previous channelNonce used by the same `participants` and `chainId`. This is to prevent states from previous channels being "replayed" to conclude subsequent channels with unintended outcomes.
+- `channelNonce` -- This should be different to any previous channelNonce used by the same `participants` and `chainId`. This is to prevent states from previous channels being "replayed" to conclude subsequent channels with unintended outcomes. Must be less than `2**48-1`.
 - `appDefinition` -- There should be a [`ForceMoveApp`]('contract-api/natspec/ForceMove') compliant contract deployed at this address, and you should have confidence that it is not malicious or suffering from security flaws. You should inspect the source code (which should be publically available and verifiable) or appeal to a trusted authority to do this.
-- `challengeDuration` -- In the extreme, this should be at least 1 block time (15 seconds on mainnet) and less than `2**48-1` seconds. In practice we recommend somewhere between 5 minutes and 5 months.
+- `challengeDuration` -- In the extreme, this should be at least 1 block time (15 seconds on mainnet) and less than `2**48-1` seconds. Whatever it is set to, the channel should be closed long before `2**48 - 1 - challengeDuration`. In practice we recommend somewhere between 5 minutes and 5 months.
```

Comment:
I'm not sure about this specific suggestion, but maybe we can have a suggestion in this vein somewhere [here](https://github.com/statechannels/statechannels/pull/3023/files#diff-68056c6c283f5e38bd767476fe7ce18e3d0bba2c14f540c5ecd9ce1456e1cedeL104-L106)?

```suggestion
- `challengeDuration` -- In the extreme, this should be at least 1 block time (15 seconds on mainnet) and less than `2**48-1` seconds. Whatever it is set to, the channel should be closed long before `2**48 - 1 - challengeDuration`. In practice we recommend somewhere between 5 minutes and 5 months. Consult the @statechannels team for more context.
```